CMakeLists.txt

```diff
@@ -122,30 +122,16 @@ endif ()
 
 
 #--------------------------------------------------------------------------------------------------
-# Extract version string and version components from Git repository.
+# Auto-generate version header.
 #--------------------------------------------------------------------------------------------------
 
-# Extract version string from Git repository.
-execute_process (
-    COMMAND ${git_command} describe --long
-    WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
-    RESULT_VARIABLE git_describe_result
-    OUTPUT_VARIABLE appleseed_version_string
-    ERROR_QUIET
-    OUTPUT_STRIP_TRAILING_WHITESPACE
-)
-
-# Use dummy version string if 'git describe' command failed (e.g. when building from a source dump).
-if (NOT ${git_describe_result} EQUAL 0)
-    set (appleseed_version_string "0.0.0")
-endif ()
-
-# Extract version components from version string.
-string (REGEX MATCHALL "[0-9]+" appleseed_version_components ${appleseed_version_string})
-list (GET appleseed_version_components 0 appleseed_version_major)
-list (GET appleseed_version_components 1 appleseed_version_minor)
-list (GET appleseed_version_components 2 appleseed_version_patch)
+set (appleseed_version_major 1)
+set (appleseed_version_minor 1)
+set (appleseed_version_patch 0)
+set (appleseed_version_alpha 24)
```

Comment:
In the absence of an "alpha" or "beta" tag, as soon as 1.x.x is reached the software is considered stable, according to the semantic versioning conventions.

This is why I stand by the proposal I outlined in our private conversation: We stick to 1.1.0 for now, with the addition of a candidate number (currently at 25). When we are ready to release our first stable version, we give it the version number 1.2.0 and we stop talking about candidate numbers (but we can and should keep them internally). From that moment on, we stick to the semantic versioning rules. So the release after 1.2.0 will be either 1.2.1, 1.3.0 or 2.0.0 depending on what it breaks.